receiver/awsxrayreceiver/Makefile

```diff
@@ -0,0 +1 @@
+include ../../Makefile.Common
```

Comment:
https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/364 starts the work to add a real receiver for xray, how does it relate to that? Can you sync up on it?

Reply:
surely I will sync up Yenlinand modify the code accordingly.